common/presenters/table/object-to-table-head.js

```diff
@@ -1,15 +1,45 @@
 const { pickBy } = require('lodash')
+const queryString = require('query-string')
 
 const i18n = require('../../../config/i18n')
 
+function buildUrl(head) {
+  const {
+    currentUrl: { pathname, query },
```

Comment:
I wonder if it would be neater to not need to pass the entire URL into this presenter, and just pass in the existing querystring. You could then use the `query-string` library to override the sort keys as necessary for each sortable column.

The url for the link can then just be the querystring: `?foo=bar&sortBy=field&sortDirection=ascending`

Reply:
Good point.